### DISCUSSION TITLE
Issue or Discussion?

### DISCUSSION THREAD

**Discussion by Mineshafter61** [Limaru-Minecraft/iciwi#65] [Announcements]
### TL;DR: Use discussions if your idea is not code-related, and use issues otherwise.

(Vanilla) Github comes with 2 ways to contact the developer for changes: Issues and Discussions. While both tabs are similar, only the Issues tab tell the developer how many requests there are. Hence, here are a few guidelines as to when you should post an Issue vs a Discussion.

**Make an Issue when you are requesting:**
- **A bug report.** I think it goes without saying that these are both important and urgent for any software, therefore these naturally go into Issues.
- **Small enhancements.** If you have any enhancements that you think can be done in a day, please create an issue and add the "Enhancement" label to your issue.
- **Medium-sized or variable-sized enhancements.** If you're unsure whether your enhancement can be done quickly, please also put it in Issues, and add the "Enhancement" and "Future" labels. This makes me aware that there are features that people want, and I should probably code them soon.

**Make a Discussion when you are requesting:**
- **Large enhancements.** These require massive changes to the code, and they shouldn't be made to constantly bug me, therefore they belong in Discussions.
- **Documentation changes.** Iciwi's documentation may be wrong sometimes, especially right after a release. However, (most) documentation is non-urgent as 99% of players won't use the advanced features of Iciwi. If you see any outdated or wrong documentation, be sure to open a Discussion.
